### PR TITLE
3.21.1 LIVE (ENG) + (GER)

### PR DESCRIPTION
**Pull request by ROBdk97**
Current State: 3.21.1 PTU 8889500
English done
German finished until 49143

### CODE REVIEW

**Comment by ROBdk97** [comment]
Current State: 3.21.1 PTU 8892745 

- [x] English 
- [x] German

**Comment by ROBdk97** [comment]
3.21.1 PTU 8892745 is the Live State of the global.ini

**Comment by ROBdk97** [comment]
I'm not sure why, but now the tables are fixed on GitHub pages.

**Comment by ROBdk97** [comment]
I am done. Ready for Merging!

**Review by Auhrus — APPROVED** [review]
Looks good from my side.

File: data/Localization/german_(germany)/global.ini [hunk 1]
```diff
@@ -42851,7 +42858,7 @@ confirmkill_desc_0007=Ich habe gehört, dass ~mission(TargetName) tot sein könn
 confirmkill_from=~mission(Contractor)
 confirmkill_location_0001=die Überreste des Schiffes
 confirmkill_location_0002=die letzte Anlaufstelle
-confirmkill_location_0003=~mission(TargetName|NickOrFirst)'s letzte bekannte Position
+confirmkill_location_0003=~mission(TargetName|Last)'s letzte bekannte Position
```
Comment (Auhrus) [inline data/Localization/german_(germany)/global.ini:42861]: Moment
Shouldn't "NickOrFirst" be left as it is?

**Review by Auhrus — CHANGES_REQUESTED** [review]
Found a little "mistake" after all. 😅

File: data/Localization/german_(germany)/global.ini (hunk 1 above)
Comment (ROBdk97) [reply data/Localization/german_(germany)/global.ini:42861]: Got changed in english version

**Pull request merged**